plugin/documents.py

```diff
@@ -545,13 +550,30 @@ def _render_code_lens(self, name: str, index: int, region: sublime.Region, comma
                 annotation = command["title"]
         else:
             annotation = "..."
-        annotation = '<div class="codelens">{}</div>'.format(annotation)
-        accent = self.view.style_for_scope("region.greenish markup.accent.codelens.lsp")["foreground"]
-        self.view.add_regions(self._code_lens_key(index), [region], "", "", 0, [annotation], accent)
+
+        show_code_lens = userprefs().show_code_lens
+        if show_code_lens == "phantom":
+            key = region.to_tuple()
+            try:
+                phantom = self._code_lens_phantom_regions[key]
+            except KeyError:
+                self._code_lens_phantom_regions[key] = phantom = CodeLensPhantom(self.view, region)
+
```

Comment:
Personally not a huge fan of empty lines. Only use empty lines when it's a function/method/closure please?